server/api/section/section.controller.js

```diff
@@ -93,6 +145,8 @@ exports.show = function(req, res) {
 
 // Creates a new Section in the DB
 exports.create = function(req, res) {
+  try{checkSectionReq(req);}
+  catch(err) { return handleError(res)(err);}
   Section.createAsync(req.body)
```

Comment:
:penguin: I'm not a huge fan of the try/catch because it's not really used anywhere else, but you did handle it properly